src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
Yeah the most commonly cited issue w/ RM for DI is exactly that.

The pushback to that sentiment is that instead of adopting a monolithic one-size-fits-all approach through the entire code base that you use specific tools which make sense for the specific purpose at various points and layers. So the more simplistic bits would be using the RM approach and you might compose a variety of those together via some other mechanism.